compiler-core/src/exhaustiveness.rs

```diff
@@ -2713,24 +2713,46 @@ impl<'a> Compiler<'a> {
 /// Returns a pattern check from `first_branch` to be used as a pivot to split all
 /// the `branches`.
 ///
-fn find_pivot_check(first_branch: &Branch, branches: &VecDeque<Branch>) -> Option<PatternCheck> {
-    // To try and minimise code duplication, we use the following heuristic: we
-    // choose the check matching on the variable that is referenced the most
-    // across all checks in all branches.
+fn find_pivot_check(
+    first_branch: &Branch,
+    branches: &VecDeque<Branch>,
+    env: &Environment<'_>,
+) -> Option<PatternCheck> {
+    // Count how often each variable is referenced across branches (old heuristic).
     let mut var_references = HashMap::new();
     for branch in branches {
         for check in &branch.checks {
             let _ = var_references
                 .entry(check.var.id)
-                .and_modify(|references| *references += 1)
+                .and_modify(|r| *r += 1)
```

Comment:
We prefer using full names instead of shorthands in the compiler, could you revert this?